lib/sentry/application.ex

```diff
@@ -71,4 +76,23 @@ defmodule Sentry.Application do
       Sentry.Integrations.Quantum.Cron.attach_telemetry_handler()
     end
   end
+
+  defp resolve_in_app_module_allow_list do
+    if Mix.Project.umbrella?() do
```

Comment:
Mix is not available in releases, so we generally can't do this at runtime. At the same time, Sentry is already compiled when the user's application compiles (as it's a dependency). So doing it at compile time won't work either.

I’m not sure we have a better way to do this other than adding a configuration, something like:

```elixir
config :sentry,
  otp_apps: [...]
```

This way users can set the config above (and figure out umbrella stuff and whatnot).